--- natmonitor-2.4/natmonitor.c.orig	2006-06-06 16:46:24.000000000 +0200
+++ natmonitor-2.4/natmonitor.c	2006-06-06 16:38:28.000000000 +0200
@@ -195,7 +195,7 @@
 	/* draw graph */
 	sprintf(string,"%s",ctime(&timevalues[t]));
 	string[strlen(string)-1]=0; /* wipe out ctime \n */
-	gdk_draw_text(pixmaplegend,font,window->style->white_gc,10+2*BORDER,ATY(0,0),string,strlen(string));
+	gdk_draw_text(pixmaplegend,font,window->style->white_gc,30+BORDER,ATY(0,0),string,strlen(string));
 
 	line=0;
 	for (i=0;i<hostnumber;i++){
@@ -211,14 +211,14 @@
 
 			if(i==0)sprintf(string,"Router: %s",hostnames[i]); else sprintf(string,"Host: %s",hostnames[i]);
 			gdk_gc_set_foreground(gc,&color[i]);
-			gdk_draw_text(pixmaplegend,font,gc,10+2*BORDER,ATY(line,1),string,strlen(string));
+			gdk_draw_text(pixmaplegend,font,gc,30+BORDER,ATY(line,1),string,strlen(string));
 
 			if(i || ROUTERONLY)totalinsample+=in[i][t];
 			string[0]=0;
 			sprintf(string,"Receiving: ");
 			DataDisplay(tmp,totalin[i]);
 			strcat(string,tmp);
-			if(!NOSUBTOTALS) gdk_draw_text(pixmaplegend,font,gc,10+2*BORDER,ATY(line,4),string,strlen(string));
+			if(!NOSUBTOTALS) gdk_draw_text(pixmaplegend,font,gc,30+BORDER,ATY(line,4),string,strlen(string));
 
 			topin=basein+in[i][t]*DSCALE/1024;
 			gdk_gc_set_foreground(gc,&colordark[i]);
@@ -226,7 +226,7 @@
 			if(i || ROUTERONLY)basein=topin;
 			gdk_gc_set_foreground(gc,&color[i]);
 			sprintf(string,"IN: %sps",DataDisplay(tmp,in[i][t]));
-			gdk_draw_text(pixmaplegend,font,gc,10+2*BORDER,ATY(line,2),string,strlen(string));
+			gdk_draw_text(pixmaplegend,font,gc,30+BORDER,ATY(line,2),string,strlen(string));
 
 
 			if(i || ROUTERONLY)totaloutsample+=out[i][t];
@@ -242,7 +242,7 @@
 			if(i || ROUTERONLY )baseout=topout;
 			gdk_gc_set_foreground(gc,&color[i]);
 			sprintf(string,"OUT: %sps",DataDisplay(tmp,out[i][t]));
-			gdk_draw_text(pixmaplegend,font,gc,10+2*BORDER,ATY(line,3),string,strlen(string));
+			gdk_draw_text(pixmaplegend,font,gc,30+BORDER,ATY(line,3),string,strlen(string));
 
 			line++;
 		}
@@ -292,12 +292,12 @@
 		gdk_draw_text(pixmaplegend,font,window->style->white_gc,10+TEXTAREA/2,ATY(line,3),string,strlen(string));
 	}
 	sprintf(string,"Total:");
-	gdk_draw_text(pixmaplegend,font,window->style->white_gc,10+2*BORDER,ATY(line,1),string,strlen(string));
+	gdk_draw_text(pixmaplegend,font,window->style->white_gc,30+BORDER,ATY(line,1),string,strlen(string));
 
 	sprintf(string,"IN : %sps",DataDisplay(tmp,totalinsample/4));
-	gdk_draw_text(pixmaplegend,font,window->style->white_gc,10+2*BORDER,ATY(line,2),string,strlen(string));
+	gdk_draw_text(pixmaplegend,font,window->style->white_gc,30+BORDER,ATY(line,2),string,strlen(string));
 	sprintf(string,"OUT: %sps",DataDisplay(tmp,totaloutsample/4));
-	gdk_draw_text(pixmaplegend,font,window->style->white_gc,10+2*BORDER,ATY(line,3),string,strlen(string));
+	gdk_draw_text(pixmaplegend,font,window->style->white_gc,30+BORDER,ATY(line,3),string,strlen(string));
 	
 	/* write reference lines and label */
 	for (i=0;i<=((MAXUPLOAD-10)  /10);i++) {
